cmd/brew-test-bot.rb

```diff
@@ -533,12 +533,17 @@ def download
           "git", "-C", @tap.path.to_s,
                  "log", "-1", "--format=%h (%s)", "origin/master"
         ).strip
-        puts "#{@tap} origin/master #{tap_origin_master_revision}"
         tap_revision = Utils.popen_read(
           "git", "-C", @tap.path.to_s,
                  "log", "-1", "--format=%h (%s)"
         ).strip
-        puts "#{@tap} HEAD #{tap_revision}"
+        puts "\nTesting tap #{@tap} with:"
+        puts "  - origin/master     - #{tap_origin_master_revision}"
+        puts "  - pull request HEAD - #{tap_revision}"
+      else
+        puts "\nTesting Homebrew/brew #{@tap} with commits:"
+        puts "    - diff_start_sha1 #{diff_start_sha1.empty? ? "(undefined)" : diff_start_sha1}"
```

Comment:
Use `.blank?` here.

Reply:
These can still be printed in addition to the other messages if they are useful and not always the same.

Reply:
I've updated the output